File: sagrada-familia-web/src/app/shared/interfaces/requests/crear-nino.request.ts
export interface CrearNinoRequest {
    representanteId: number;
    nombre: string;
    apellido: string;
    fechaNacimiento: string;
    sexo: string;
}

export interface ActualizarNinoRequest {
    nombre: string;
    apellido: string;
    fechaNacimiento: string;
    sexo: string;
}

File: SagradaFamilia/SagradaFamilia.Application/DTOs/Ninos/CrearNinoRequest.cs
namespace SagradaFamilia.Application.DTOs.Ninos
{
    public class CrearNinoRequest
    {
        public int RepresentanteId { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string Apellido { get; set; } = string.Empty;
        public DateOnly FechaNacimiento { get; set; }
        public char Sexo { get; set; }
    }
}
